components/brave_wallet_ui/reducers/index.ts

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { combineReducers } from 'redux'
+
+// Utils
+import walletPanelReducer from './wallet_panel_reducer'
+
+export default combineReducers({
+  walletPanelReducer
```

Comment:
I forgot about this small issue, we can do as a follow-up. It’s a bit strange to name this `walletPanelReducer` as the keys in this object are the property keys in the state. Seems ok for now to call it something like just `walletPanel` if we want to still use `combineReducers` even though we only have 1 reducer, for future proofing.